packages/common/src/intl/locales/en/common.json

```diff
@@ -156,5 +158,6 @@
   "label.verified": "Verified",
   "link.backorders": "Backorders",
   "link.copy-to-clipboard": "Copy to Clipboard",
-  "link.history": "History"
-}
\ No newline at end of file
+  "link.history": "History",
+  "messages.this-batch-has-expired": "This batch has expired."
```

Comment:
oops. not used yet - taken from #1053

Reply:
I though it was a cute idea. Though, I realised after adding and hadn't finished yet that the original `red` was coming from 'almost expired'.

Had a quick look at some options of showing 'this batch expires in X days' or something similar but there was nothing easy. So.. thought to do something like 'expiring soon'/'expired' 🤷